src/modals/select-lane-modal.tsx

```diff
@@ -0,0 +1,62 @@
+import { useQueryClient } from "@tanstack/react-query";
+import { Text } from "ink";
+import { useAtomValue } from "jotai";
+import { useGetIssueTransitionsQuery } from "../api/get-issue-transitions.query.js";
+import type { Issue } from "../api/get-issues.query.js";
+import { useTransitionIssueMutation } from "../api/transition-issue.mutation.js";
+import { highlightedIssueAtom } from "../atoms/highlighted-issue.atom.js";
+import type { Option } from "./select-modal.js";
+import { SelectModal } from "./select-modal.js";
+
+export const SelectLaneModal = ({
+  onClose,
+}: {
+  onClose: () => void;
+}) => {
+  const { mutate: transitionIssue } = useTransitionIssueMutation();
+  const highlightedIssue = useAtomValue(highlightedIssueAtom);
+  const queryClient = useQueryClient();
+  const issues = queryClient.getQueryData(["issues"]) as Issue[];
+
+  const issue = issues.find((issue) => issue.id === highlightedIssue.id)!;
+
+  const { data: transitions } = useGetIssueTransitionsQuery(
+    issue?.id!,
+    !!issue,
+  );
+
+  if (!transitions || transitions.length === 0) {
+    return <Text>Loading...</Text>;
+  }
+
+  const selectedTransition = transitions.find(
+    (transition) => transition.to.id === issue.fields.status.id,
+  );
+
+  return (
+    <SelectModal
+      options={transitions.map((transition) => ({
+        label: transition.to.name,
+        value: transition.id,
+        extraData: {
+          newStatusId: transition.to.id,
+        },
+      }))}
```

Comment:
To investigate: how to allow user-provided sorting for these lanes (maybe just config for now).

It's really hard to use this in a normal dev flow, the ordering is all out of order (compared to the lanes on the actual board view)

<img width="369" alt="afbeelding" src="https://github.com/user-attachments/assets/9fd14a68-17bf-48da-bf5c-dc0f4eac723d" />